r-seurat-scripts-post-install-tests.bats

```diff
@@ -121,14 +121,27 @@
     [ -f  "$pca_seurat_object" ]
 }
 
+
+#Run transfer anchor
+@test "Find transfer anchors" {
+      if [ "$use_existing_outputs" = 'true' ] && [ -f "$anchor_object" ]; then
+          skip "$anchor_objet exists and use_existing_outputs is set to 'true'"
+      fi
+
+      run seurat-find-transfer-anchor.R -i $pca_seurat_object -r $pca_seurat_object -o $anchor_object --normalization-method LogNormalize --dims 1:10  
+      echo "status = ${status}"
+      echo "output = ${output}"
+      [ "$status" -eq 0 ]
+}
+
 # Find Neighbours
 
 @test "Run FindNeighbours" {
     if [ "$use_existing_outputs" = 'true' ] && [ -f "$neighbours_seurat_object" ]; then
         skip "$neighbours_seurat_object exists and use_existing_outputs is set to 'true'"
     fi
 
-    run rm -rf $neighbours_seurat_object && seurat-find-neighbours.R -i $pca_seurat_object -o $neighbours_seurat_object --dims 1,2,3,4,5 --compute-snn --reduction pca
+    run rm -rf $neighbours_seurat_object && seurat-find-neighbours.R -i $pca_seurat_object -o $neighbours_seurat_object --compute-snn --reduction pca
```

Comment:
Why have you changed the call to find-neighbours?

Reply:
Sorry it was not supposed to be here